recaf-core/src/main/java/me/coley/recaf/parse/JavaParserHelper.java

```diff
@@ -165,47 +169,49 @@ private boolean isInvalidCompilationUnit(CompilationUnit unit) {
 	 *     <li>{@link me.coley.recaf.code.MethodInfo}</li>
 	 * </ul>
 	 */
+
 	public Optional<ParseHitResult> at(CompilationUnit unit, int line, int column) {
 		WorkspaceTypeSolver typeSolver = symbolSolver.getTypeSolver();
-		if (unit != null) {
-			Node node = getNodeAtLocation(line, column, unit);
-			while (node != null && JavaParserResolving.isNodeResolvable(node)) {
-				ItemInfo value = JavaParserResolving.of(symbolSolver, node);
-				if (value == null) {
-					Optional<Node> parent = node.getParentNode();
-					if (parent.isPresent()) {
-						// Check if the parent item can be resolved
-						node = parent.get();
-						continue;
-					} else {
-						// No parent, end the loop
-						break;
-					}
-				} else if (node instanceof SimpleName && value instanceof FieldInfo) {
-					// In some obfuscated cases, a method name may be selected as an AST 'SimpleName'.
-					// These typically resolve to fields of matching names.
+		if (unit == null) return Optional.empty();
+		Node node = getNodeAtLocation(line, column, unit);
+		if (node instanceof Expression) {
+			Expression expression = (Expression) node;
+			// Double does not have any conversions yet
+			if (!expression.isDoubleLiteralExpr()) {
```

Comment:
`evaluate(LiteralExpr literalExpr)` seems to have support, unless I'm misinterpreting

Reply:
Yes, but only expressions, not simple literals.

Reply:
Ah gotcha